generic_config_updater/gu_common.py

```diff
@@ -691,3 +693,44 @@ def _get_model(self, model, name):
                     return submodel
 
         return None
+
+class GenericUpdaterLogger:
```

Comment:
>GenericUpdaterLogger [](http://example.com/codeflow?start=6&length=20)

This class seems have general value, like [https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/logger.py](https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/logger.py).

Is it better to enhance existing Logger class? If something is impossible, we can inherit the class.

Reply:
used common logger